doc/nrf/security/ap_protect.rst

```diff
@@ -59,6 +59,11 @@ See the related hardware documentation for more information about which implemen
      - Hardware and software AP-Protect support
      - Related hardware documentation
      - Additional information
+   * - nRF9161
+     - ✔
+     -
+     - `Access protection for nRF9161`_
+     - Also supports Secure AP-Protect (see note below)
```

Comment:
@anangl , can you please confirm this information is correct for nRF9161? Thanks.

Reply:
Please note this was confirmed by @frkv

Reply:
You mean confirmed that the procedures for enabling AP-Protect for nRF9161 are matching the contents of the sections below this table?

I'd like @anangl to still have a look, as he co-authored this page with Simen back in the days.

Reply:
I meant that it is confirmed that both 9160 and 9161 have support for APPROTECT and SECUREAPPROTECT. 

Sure, we can have Andrzej's input also in this.

Reply:
Yes, both have APPROTECT and SECUREAPPROTECT, but the AP-Protect implementation type is different.

Reply:
@anangl does this mean the procedures for enabling AP-Protect https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/security/ap_protect.html#configuration-overview-in-the-ncs are _not_ applicable to nRF9161?

Reply:
No, the procedures are applicable. Only the type of AP-Protect implementation indicated by that checkmark above is specified incorrectly. As I also mentioned in https://github.com/nrfconnect/sdk-nrf/pull/13744#discussion_r1476132189, it should be "Hardware and software" for nRF9161,

Reply:
Thanks, @anangl !